sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ConnectionPolicy.java

```diff
@@ -476,10 +458,9 @@ public String toString() {
             ", inetSocketProxyAddress=" + inetSocketProxyAddress +
             ", readRequestsFallbackEnabled=" + readRequestsFallbackEnabled +
             ", connectionTimeout=" + connectionTimeout +
-            ", idleChannelTimeout=" + idleChannelTimeout +
             ", idleEndpointTimeout=" + idleEndpointTimeout +
-            ", maxChannelsPerEndpoint=" + maxChannelsPerEndpoint +
-            ", maxRequestsPerChannel=" + maxRequestsPerChannel +
+            ", maxChannelsPerEndpoint=" + maxConnectionsPerEndpoint +
+            ", maxRequestsPerChannel=" + maxRequestsPerConnection +
```

Comment:
We should also rename the string wordings -> `maxChannelsPerEndpoint` -> `maxConnectionsPerEndpoint`
and `maxRequestsPerChannel` -> `maxRequestsPerConnection`

Reply:
another good catch. change made on my branch. to be committed with this PR.